doc/api/http.md

```diff
@@ -1290,7 +1290,11 @@ event is emitted on the server object, and (by default) the socket is destroyed.
 See [`server.timeout`][] for more information on how timeout behavior can be
 customized.
 
-### `server.listen()`
+### `server.listen(port[, host][, callback])`
```

Comment:
Please keep it as is. `server.listen()` has multiple signatures. See https://nodejs.org/api/net.html#net_server_listen.